addons/arsenal/script_component.hpp

```diff
@@ -14,4 +14,21 @@
     #define DEBUG_SETTINGS DEBUG_SETTINGS_ARSENAL
 #endif
 
+#ifdef DISABLE_COMPILE_CACHE
+    #define RECOMPILE
+#endif
+
 #include "\z\ace\addons\main\script_macros.hpp"
+
+
+// if use just GVAR here, "Undefined variable" error occurs
+#define WAIT_AND_CHECK_SETTING(func) \
+    if (missionName isEqualTo 'Arsenal') exitWith {_this call DOUBLES(BIS_fnc,func)BIS}; \
```

Comment:
call BIS_fnc_arsenalBIS ? The BIS at the end looks weird.

Reply:
what would you suggest to use as prefix/suffix? Because `BIS_fnc_arsenal` is already wrapper.

Reply:
Ah understand now. Btw the ACE Arsenal mission is also called `Arsenal` But I guess you don't call BIS_fnc_arsenal in there